lhotse/augmentation/torchaudio.py

```diff
@@ -0,0 +1,111 @@
+import random
+import warnings
+from dataclasses import dataclass
+from typing import List, Union
+
+import numpy as np
+import torch
+import torchaudio
+
+if str(torchaudio.__version__) < '0.7.0':
+    warnings.warn('Torchaudio SoX effects chains are only introduced in version 0.7 - '
+                  'please upgrade your PyTorch to 1.7+ and torchaudio to 0.7+ to use them.')
+
+
+@dataclass
+class RandomValue:
+    """
+    Represents a uniform distribution in the range [start, end].
+    """
+    start: Union[int, float]
+    end: Union[int, float]
+
+    def sample(self):
+        return random.uniform(self.start, self.end)
```

Comment:
can we use numpy random functions? It may be easier for me to seed it from outside

Reply:
Would using this function help (note that it also makes the random cut ID creation deterministic)? 

https://github.com/lhotse-speech/lhotse/blob/master/lhotse/utils.py#L33

Reply:
OK, never mind. I will seed it in my own code.

Reply:
I'll change it to numpy anyway, I guess more people are used to seeding `numpy` than `random` in training loops